src/main/java/com/dev/cinema/service/mapper/CinemaHallMapperImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapperImpl implements CinemaHallMapper {
```

Comment:
we don't really need interfaces for these mappers - let's remove the interfaces
```suggestion
public class CinemaHallMapper {
```